packages/docusaurus-theme-classic/src/theme/BackToTopButton/styles.module.css

```diff
@@ -18,19 +13,33 @@
   right: 1.3rem;
   bottom: 1.3rem;
   border-radius: 50%;
-  background: var(--docusaurus-btt-background);
-  color: var(--docusaurus-btt-color);
+  background-color: var(--ifm-color-secondary);
   width: 3rem;
   height: 3rem;
   z-index: var(--ifm-z-index-fixed);
-  box-shadow: 0 0.125rem 0.3125rem 0 rgba(0, 0, 0, 0.3);
-  transition: all var(--ifm-transition-fast) ease-in-out;
+  box-shadow: var(--ifm-global-shadow-lw);
+  transition: all var(--ifm-transition-fast)
+    var(--ifm-transition-timing-default);
   opacity: 0;
   transform: scale(0);
 }
 
-.backToTopButton:not(:focus):hover {
-  opacity: 0.8;
+.backToTopButton:after {
+  content: ' ';
+  mask: var(--ifm-menu-link-sublist-icon) 50% / 2rem 2rem no-repeat;
```

Comment:
Why use mask and background-color, maybe just `background: var(--ifm-menu-link-sublist-icon) 50% 50% / 2rem 2rem no-repeat;` instead?

Reply:
> Perhaps we need to add dark mode support also?

![image](https://user-images.githubusercontent.com/1576283/132520911-2c177461-88fa-4644-8e74-b38984d12a3d.png)

Better if I use the emphasis colours instead of secondary so it swaps colours?

Using mask allows you to directly re-colour the svg by just setting a background-color, and not having to edit underlying embedded svg or mess with filters like infima is doing for the sidebar arrows in light vs dark. The sidebar menu that uses this same icon should be made to do the same, but that code is on the infima side I think.

Reply:
Interesting idea with using mask, although it requires vendor prefix to work. Maybe it makes sense to use mask instead of filter in Infima..

Agree about using emphasis colors, but in dark mode the button background is pretty bright, maybe use 200 instead of 300? Then the button would look like collapse sidebar button by color. WDYT?

| Before   | After    |
| -------- | -------- |
| ![image](https://user-images.githubusercontent.com/4408379/132530736-5a9bc3d3-530d-41f1-8d52-8fb384acf75b.png) | ![image](https://user-images.githubusercontent.com/4408379/132531546-9aaff6d1-6778-4641-9ec0-ddd1fe08a68f.png) |

Reply:
![image](https://user-images.githubusercontent.com/1576283/132576949-47cdde12-8fd5-442b-ba35-eecb9816ccc2.png)

Stepped down to 200.  Cool now?

The collapse bar is not part of the color scale? I've never noticed, I have it turned off.

Reply:
Yes, it looks great now.

Collapse button uses the built-in button classes from Infima, `button button-secondary', to which dark mode support has been added in Docusaurus code. Perhaps we should do the same with back-to-top button?

Reply:
Sorry, is there something else you want changed in this PR or is it good to go?

Reply:
Yes, your PR is fine, just added the new message to translate in supported languages and removed unnecessary CSS code.

Reply:
I also removed `active` state you added, because for now it actually does not work -- we need to add `!important` rule for this, but I think this state is not too important for such small button.